python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -258,8 +304,29 @@ keys:
                       convert_types:
                         - str
                       description: Network inner VLAN ID.
+                dot1ad:
+                  type: dict
+                  keys:
+                    vlan:
+                      type: int
+                      convert_types:
+                        - str
+                      description: Network VLAN ID.
+                    outer:
+                      type: int
+                      convert_types:
+                        - str
+                      description: Network outer VLAN ID.
+                    inner:
+                      type: int
+                      convert_types:
+                        - str
+                      description: Network inner VLAN ID.
                 client:
                   type: bool
+                untagged:
+                  type: bool
+                  description: No encapsulation.
```

Comment:
These different bool can not be set at the same time correct?
So I think we should make it a string holding the different options. Maybe we can combine with the protocols above or something else.
The idea is to make the data model implicitly enforce that only one of the values can be set.

Reply:
yes, they cannot be set at the same time.

Reply:
Updated the schema